sdk/python/kfp/client/__init__.py

```diff
@@ -13,3 +13,9 @@
 # limitations under the License.
 
 from kfp.client.client import Client
+
+from kfp.client.token_credentials_base import TokenCredentialsBase, read_token_from_file
```

Comment:
It looks to me that `read_token_from_file` doesn't need to be exposed from `kfp.client`.